### ISSUE TITLE
express compression problem

### ISSUE THREAD

**Issue by eajimmy** [Nubisa/jxdocs#168]
Hi, thank you for your great job with JXcore!
Unfortunately I'm having an issue with the 'compression' module used from express.
The issue is the following:
if the module is required in the express index.js (app.use(require('compression')());), the files in public directory (.js, css, but not the images) are not serving from the webserver, especially the ones with dimension greater than 1KB.
Instead, the browser keep loading and the requests remain pending.
In my opinion it happens due to the method the compression module uses for retrieve files from filesystem: i.e. with a stream, chunk by chunk. Is this a possible explanation? Are there some workaround/fixes?
I try removing the 'compression' require or leave the public folder outside the .jx file, both methods work fine, but they are not great solutions (due to the lack of performance and secuirty).

**Comment by obastemur** [comment]
Could you please submit a test case? We are close to a release and I would like to fix this one before the release.

**Comment by eajimmy** [comment]
At this url you will find a little project for istance, ready to be jxcore'd:
https://mega.co.nz/#!ZlZBDCYI!CQXsA7GP8JyjzhjMkjcBfMFCVNgYrmC1HfWQ8kXPE3g
in webserver/index.jx there is the line with the require to toggle.
Remeber to refresh browser cache between the 2 test (with and without the comment);

**Comment by eajimmy** [comment]
Hi obastemur, do you have any news/updates/fixes about this issue?

**Comment by obastemur** [comment]
@eajimmy thanks for the package. Actually I've just tested it with the master build and it works.

  - Downloaded your package and unpacked
  - inside the main folder. 'jx install express' and 'jx install ejs-local'
  - finally 'jx package webserver/index.js my_package' 
  - jx my_package.jx

**Comment by eajimmy** [comment]
@obastemur, yes, it works as is, but can you please try to decomment the line 16 in index.js (enabling the compression module), repackage all and try if it works? Thank you.

**Comment by ktrzeciaknubisa** [comment]
@eajimmy Which exactly express version are you using?

**Comment by eajimmy** [comment]
@ktrzeciaknubisa the versions I have tried are:
- express: 4.11.1
- compression: 1.0.1
